purrfect-match/src/ContactPage.tsx

```diff
@@ -3,6 +3,9 @@ import './ContactPage.css';
 import { Input, Select, Message, Send } from './inputBoxes';
 import { Api } from "Api";
 import { Modal } from "Modal";
+import { string } from 'prop-types';
```

Comment:
Do we really need this? At least import it under a different name to avoid confusing name collisions if necessary.